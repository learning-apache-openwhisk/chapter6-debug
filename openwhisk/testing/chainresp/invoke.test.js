const ow = require('openwhisk')

test("invoke email validation", () => {
    ow()
      .actions.invoke({
        name: "testing/strategy-email",
        params: {
          email: "michele.sciabarra.com"
        }
      }).then(res => 
        expect(res).toMatchSnapshot())
}) 
