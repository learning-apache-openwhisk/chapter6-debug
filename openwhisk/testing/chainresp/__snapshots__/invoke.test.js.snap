// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`invoke email validation 1`] = `
Object {
  "email": "michele.sciabarra.com",
  "errors": Array [
    "michele.sciabarra.com is not an email address",
  ],
  "message": Array [],
}
`;
